source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,263 @@
+You can use the [GOV.UK skip link component](https://design-system.service.gov.uk/components/skip-link/)
+for this.
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+### Sectioned content
 
-### Emphasis
+Use the `<section>` tag to represent standalone sections where a more specific
+semantic element would not be appropriate.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <nav role="navigation">
-      <ol>
-        <li><a href="/">Home</a></li>
-        <li><a href="/business">Business</a></li>
-        <li><strong>Business Rates</strong></li>
-      </ol>
-    </nav>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
+
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
+
+### Navigational elements
+
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). As above, it is good to use `aria-labelledby`
+to disambiguate if you have multiple `<nav>` blocks, which should generally
+be used quite sparingly.
+
+The detail of using the `<nav>` tag is covered in this [GOV.UK blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links, related content, and advertisements.
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
+
+`<b>` and `<i>` tags make text bold and italic but have no semantic meaning; it
+is purely for visual styling.
+
+`<strong>` and `<em>` have the same effect visually, but have semantic meaning.
+In other words, the words they encapsulate are more important than the surrounding
+words.
+
+An example usage of `<strong>` is if we want to show the user their current location
+within a navigational list.
+
+```html
+<nav role="navigation">
+  <ol>
+    <li><a href="/">Home</a></li>
+    <li><a href="/business">Business</a></li>
+    <li><strong>Business Rates</strong></li>
+  </ol>
+</nav>
+```
+
+### Images
```

Comment:
I'm uncertain about this section, because I think this document should both reflect what GDS does and also state best practice - but I'm not sure what GDS does with images (GOV.UK doesn't have a lot of them and I don't think we use anything much more complicated than basic img tags). Worth a discussion.

Reply:
Agreed, it felt a bit out of scope as I was writing it. Tried balancing best practice with what's actually required on GDS sites and this seemed a bit unnecessary.

Reply:
It'd be worth seeing what @alex-ju thinks with his experience on Content Publisher and how images are used in that.

Reply:
I've cut this section right down, but may be worth cutting down even further if we don't want to go into the merits of webp and srcset in this document. Could you give this a look @alex-ju ?

Reply:
Have had a think and have now removed the guidance on format and responsiveness - it's something we can add in future if we need it.

Image section now clarifies `alt` text responsibility and nothing else.

Reply:
We have a big problem with alt texts, most people don't understand how it should be written. (And many experts don't even agree on rules.) But the GOV.UK content style guide and/or the Design System are probably the better place to have that kind of information. In the end it's much more related to content than to code.